lua/player/init.lua

```diff
@@ -100,25 +108,52 @@ M.setup = function(opts)
          end
 
          if not is_playback_command(arg2) then
-            return notify("Invalid player argument " .. arg2, "WARN")
+            return notify("Invalid player argument " .. arg2, vim.log.levels.WARN)
          end
 
-         system("playerctl -p " .. arg1 .. " " .. arg2)
-         return notify_player(arg1)
+         M.run_player_command(arg1, arg2)
       end
 
       if arg1 ~= "" and not is_supported_player(arg1) and not is_playback_command(arg1) then
-         return notify("Invalid argument " .. arg1, "WARN")
+         return notify("Invalid argument " .. arg1, vim.log.levels.WARN)
       end
 
-      system("playerctl " .. arg1)
-      notify_player()
+      M.run_command(arg1)
    end, {
       nargs = "*",
       complete = function()
          return player_args
-      end
+      end,
    })
 end
 
+--- Run command on the specific player and notify the user
+---@param command string
+---@param player string
+---@return nil
+M.run_player_command = function(command, player)
+   system("playerctl -p " .. player .. " " .. command)
+   return notify_player(command)
+end
+
+--- Run command with playerctl and notify the user
+---@param command string
+---@return nil
+M.run_command = function(command)
+   system("playerctl " .. command)
+   return notify(command)
+end
+
+--- Show a UI selection for playback commands
+---@param select_opts table Options for vim.ui.select
+---@return nil
+M.select = function(select_opts)
```

Comment:
I don't see the need for this especially if the available actions are already listed in the cmdline completion